ironfish/src/network/testUtilities/helpers.ts

```diff
@@ -94,6 +94,25 @@ export function getConnectedPeer(
   return { peer, connection: connection }
 }
 
+export function getDisconnectedPeer(
+  pm: PeerManager,
+  identity?: string | Identity,
+): { peer: Peer; connection: WebSocketConnection } {
+  const { peer, connection } = getConnectingPeer(pm)
+
+  if (!identity) {
+    identity = jest.requireActual<typeof import('uuid')>('uuid').v4()
+  }
+
+  if (!isIdentity(identity)) {
+    identity = mockIdentity(identity)
+  }
```

Comment:
The contents of this function can be replaced with `pm.getOrCreatePeer(null)`, since a new peer is disconnected by default. Likewise, it doesn't really make sense to return a connection from this function, because a Peer doesn't have any connections on itself when it's in the disconnected state.